pyproject.toml

```diff
@@ -71,8 +71,7 @@ xattr = { version = "^0.9.7", markers = "sys_platform == 'darwin'" }
 urllib3 = "^1.26.0"
 dulwich = "^0.20.44"
 
-[tool.poetry.dev-dependencies]
-tox = "^3.18"
+[tool.poetry.group.test.dependencies]
```

Comment:
I don't like just changing labels like this. If we want to make use of groups, we should split the deps properly (mypy and pre-commit are not "test" deps)

Reply:
That was a brainfart -- it was supposed to be dev